plugins/tweak/tweaks/material-size-for-melting.h

```diff
@@ -0,0 +1,127 @@
+#include "modules/Materials.h"
+
+#include "df/inorganic_raw.h"
+#include "df/item_constructed.h"
+#include "df/item_armorst.h"
+#include "df/item_glovesst.h"
+#include "df/item_helmst.h"
+#include "df/item_shoesst.h"
+#include "df/item_pantsst.h"
+#include "df/item_weaponst.h"
+#include "df/item_shieldst.h"
+#include "df/item_toolst.h"
+#include "df/item_trapcompst.h"
+#include <cmath>
+#include <modules/Random.h>
+
+
+static Random::MersenneRNG rng;
+static bool init_rng = true;
+
+static float get_random() {
+    if (init_rng) {
+        rng.init();
+        init_rng = false;
+    }
+    return static_cast <float> (rng.drandom1());
+}
```

Comment:
Really don't like this part, but with my very limited knowledge of C++ I have no other idea how this should be handled.